holidays/countries/andorra.py

```diff
@@ -49,90 +102,100 @@ def __init__(self, *args, **kwargs) -> None:
 
     def _populate_public_holidays(self) -> None:
         # New Year's Day.
-        self._add_new_years_day("New Year's Day")
+        self._add_new_years_day(tr("Cap d'Any"))
 
         # Epiphany.
-        self._add_epiphany_day("Epiphany")
+        self._add_epiphany_day(tr("Reis"))
 
         # Carnival.
-        self._add_carnival_tuesday("Carnival")
+        self._add_carnival_monday(tr("Carnaval"))
 
-        # Constitution Day.
-        self._add_holiday_mar_14("Constitution Day")
+        if self._year >= 1994:
+            # Constitution Day.
+            self._add_holiday_mar_14(tr("Dia de la Constitució"))
 
         # Good Friday.
-        self._add_good_friday("Good Friday")
+        self._add_good_friday(tr("Divendres Sant"))
 
-        # Easter Sunday.
-        self._add_easter_monday("Easter Monday")
+        # Easter Monday.
+        self._add_easter_monday(tr("Dilluns de Pasqua"))
 
         # Labor Day.
```

Comment:
I'm not really sure about leaving Andorra without a `start_year` parameter due to Labour Day's existence here - maybe 1934 wouldn't be too bad since their previous constitution was adopted in 1933. 👀